### PR TITLE
Add accimage python package

### PR DESCRIPTION
**Pull request by willprice**
Add the accimage Pillow-API compatible package (https://github.com/pytorch/accimage)

### CODE REVIEW

**Comment by willprice** [comment]
Hi @conda-forge/help-c-cpp
This package depends on Intel IPP which is packaged on the Intel conda channel: https://anaconda.org/intel/ipp
I understand that dependencies outside conda-forge are heavily discouraged, and have read through the [tips](https://conda-forge.org/docs/recipe.html#avoid-dependencies-outside-of-conda-forge), however I'm unsure whether I'm unable to package Intel IPP on conda-forge.

Intel IPP is licensed under the [Intel Simplified Software License](https://software.intel.com/en-us/license/intel-simplified-software-license) which has the following clauses in it:

> * Redistributions must reproduce the above copyright notice and the following terms of use in the Software and in the documentation and/or other materials provided with the distribution.
> * Neither the name of Intel nor the names of its suppliers may be used to endorse or promote products derived from this Software without specific prior written permission.
> * No reverse engineering, decompilation, or disassembly of this Software is permitted.

I believe this means that I should be OK packaging, but before going to the effort, is there any precedence for packaging these libraries in conda-forge? The best I could find is https://github.com/conda-forge/staged-recipes/issues/3022 which seems to suggest that packaging Intel MKL as advised by NumFOCUS legal council.

**Comment by willprice** [comment]
Alternatively is there any possibility of depending on those packages from the `intel` channel?

**Comment by beckermr** [comment]
Let’s bump @conda-forge/core on this one.

**Comment by scopatz** [comment]
If it is liscenced under ISSL, then you are allowed to make a conda-forge package for it. It seems like you should make an IPP package first.

**Comment by willprice** [comment]
@conda-forge/staged-recipes could you review this when you have time please?

Many thanks :)

File: recipes/accimage/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+{% set name = "accimage" %}
+{% set version = "0.1" %}
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/pytorch/accimage.git
+  git_rev: '1e81f95'
```
Comment (scopatz) [inline recipes/accimage/meta.yaml]: Please do not use repos to get sources: https://conda-forge.org/docs/meta.html#build-from-tarballs-not-repos

Comment (willprice) [reply recipes/accimage/meta.yaml]: I would have used a tarball if one is provided, however sadly there aren't any formal releases of accimage. Is this a deal breaker, or is it acceptable? I can try and get the release tagged upstream

Comment (scopatz) [reply recipes/accimage/meta.yaml]: This is pretty much a deal breaker if the project doesn't even have a version tagged.  GitHub will generate a tarball for each tag, so if accimage had a tag you could use that.  The reason that this is problematic is that we can't be sure that the project won't release a v0.1 in the future, which will then conflict with this package.

Comment (willprice) [reply recipes/accimage/meta.yaml]: Fair enough. I'll try and get upstream to tag it and report back once that's happened.

Comment (scopatz) [reply recipes/accimage/meta.yaml]: Thanks!

Comment (willprice) [reply recipes/accimage/meta.yaml]: OK, tagged and switched to using the github release tarball :+1:

**Pull request merged**